src/lib/WeatherVane/WeatherVane.hpp

```diff
@@ -45,20 +45,21 @@
 #include <px4_module_params.h>
 #include <matrix/matrix/math.hpp>
 
+#define MASK_MAN_EN 		(1<<0)	// weathervane enabled for manual position control
```

Comment:
@dagar This is not final yet as I'm still not 100% sure how configurable we need to have it. What's missing for sure is to disable weathervane completely in auto mode.

Reply:
Then maybe we should discuss the use cases? What are the reasons it would need to be configured with this level of granularity?